packages/astro/client.d.ts

```diff
@@ -81,6 +81,7 @@ declare module 'astro:assets' {
 }
 
 type ImageMetadata = import('./dist/assets/types.js').ImageMetadata;
+type SvgComponent = import('./dist/assets/types.js').SvgComponent;
```

Comment:
Otherwise the type would be global, which is confusing IMO. Check my next suggestion
```suggestion
```